gatsby_fp/src/components/Layout/Head.js

```diff
@@ -0,0 +1,16 @@
+import React from "react"
+import Helmet from "react-helmet"
+
+export default function Head({ pageTitle }) {
+  return (
+    <>
+      <Helmet
```

Comment:
What is the value of a component that does nothing more than return another component? Should just be able to call this Helmet component each place that you currently call the Head component and then remove the Head component, no?

Reply:
Although Helmet is not a component (its a plugin), we could get rid of the Head component if you prefer.  I made it for the same reason as the new Body component, so it is not written twice.  (It is used in Layout.js and 404.js). It's just a way smaller bit of code as compared to the Body component @mf-stevens

Reply:
Strictly speaking, it actually is a component—it's just one that modifies the `<head>` of a document.   :>  Nested and latter components will override duplication rather than render twice, if you find it useful.

See https://github.com/nfl/react-helmet

Reply:
Oops sorry! It makes sense that it’s a JSX component and also a plug-in, I was under the impression @mf-stevens was referring to helmet as one of our own components.  Therefore now I do not see the benefit to having the head component and I will remove it.  Thanks for the help @davidwatson

Reply:
Ah, ok. I guess it's nice having this Head component to eliminate duplication of specifying `bodyAttributes` every time we want the `title` to be different. The amount of saved duplication is quite brief right now, but might be larger in the future, so we'll keep it.

Reply:
Hah! Too bad github doesn't let you know when someone else is typing a reply... I'm good with either way.

Reply:
@mf-stevens Oh okay sounds good, keeping the component seems like the way to go then , especially if it has potential to grow in the future

Reply:
@davidwatson I gave the initial comment on each of these changes a thumbs up to indicate that they were all addressed.  For this specific change, Mike and I agreed to keep the component, how do you feel about this?

Reply:
If you're both good with it as-is for now, that's fine--we can refactor later if warranted. Just know that you can reuse `<Helmet>` to override as many times as you need to in child components and the like.

Reply:
Thank you, useful information!  @davidwatson